ui/src/shared/components/ThresholdSetting.tsx

```diff
@@ -57,7 +57,7 @@ const ThresholdSetting: FunctionComponent<Props> = ({
   } else if (type === COLOR_TYPE_MAX) {
     label = 'Maximum'
   } else {
-    label = 'Value is <='
+    label = 'Value is >='
```

Comment:
this change more accurately reflects the functionality of setting the thresholds